scripts/script_utils.py

```diff
@@ -114,30 +118,76 @@ def get_cache_service(manager):
 
 
 def get_rules(args) -> List[dict]:
```

Comment:
I think this new logic should be more like:
```python
def get_rules(args) -> List[dict]:
    return (
        load_rules_from_local(args) if args.local_rules else load_rules_from_cache(args)
    )
```

`load_rules_from_cache` will pretty much be the same as `get_rules` was before your updates
`load_rules_from_local` needs logic to handle args.rules (if it does/doesn't exists), args.standard, and args.version